TombEngine/Renderer/RendererEnums.h

```diff
@@ -83,6 +82,8 @@ constexpr auto SHININESS_VERTEX_SHIFT = 16;
 constexpr auto LOCKED_VERTEX_SHIFT = 24;
 constexpr auto INDEX_IN_POLY_VERTEX_SHIFT = 25;
 
+#define SAFE_DELETE(x) if (x != nullptr) x.reset();
```

Comment:
This macro does not belong to Renderer**Enums**.h. Move it to appropriate place.

Reply:
Done